docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+# This file is used to run the Eidolon server, a MongoDB database, and the Eidolon UI together in a single command.
+# Run `docker-compose up` in the same directory as this file to start the services.
+version: '3.8'
+
+services:
+  mongo:
+    image: mongo:latest
+    ports:
+      - "27017:27017"
+    volumes:
+      - mongo-data:/data/db
+
+  eidolon-server:
+    build:
+      context: ./examples
+      dockerfile: Dockerfile
+      args:
+        - EIDOLON_VERSION=latest
```

Comment:
This will not build the right version unless the latest version has been built locally before hand. Since we are just using the chatbot (which does not have any additional dependencies) It might be worth depending on the dockerfile in sdk. Alternatively, this is probably fine for now if we just add the sdk build to the script

Reply:
I managed to switch to the SDK Dockerfile without a problem. I added the same Dockerfile change as the other:
```
RUN poetry export --without dev --without-hashes --format=requirements.txt | sed '/^-e /d' > dist/requirements.txt
```